rulebook/synonym_rules.yaml

```diff
@@ -61,6 +61,28 @@ rules:
       - input: regulation of immunoglobulin biosynthetic process
         output: regulation of immunoglobulin biosynthesis pathway
         ontology: GO
+  - type: NewSynonym
+    branches:
+      - biological_process
+    match: "(.*[^r])ase activity"
+    match_scope: "*"
+    replacement: "\\1ation"
+    replacement_scope: exact
+    tests:
+      - input: methylase activity
+        output: methylation
+        ontology: GO
+  - type: NewSynonym
+    branches:
+      - biological_process
+    match: "(.*)rase activity"
```

Comment:
this is a bit too general - e.g transferase->transferization doesn't make sense

lt's hardcode polymerase->polymerization for now

Reply:
oh i see we have that below...

Reply:
ok, I think this is good, we have to expect a few nonsensical generated synonyms, it's oK as they won't actually be used in text